test/binaryen.js/kitchen-sink.js

```diff
@@ -959,6 +961,18 @@ function test_for_each() {
     assert(expected_data[i] === str);
   }
 
+  var constExprRef = module.i32.const(0);
+  module.setFunctionTable(1, 0xffffffff, funcNames, constExprRef);
```

Comment:
It's strange that `setFunctionTable` only supports a single segment, but I don't think this PR is the right place to change that. Can you add a TODO pointing out this deficiency?